modules/cratedb/testcontainers/cratedb/__init__.py

```diff
@@ -0,0 +1,158 @@
+import os
+import typing as t
+from urllib.parse import quote
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.exceptions import ContainerStartException
+from testcontainers.core.utils import raise_for_deprecated_parameter
+from testcontainers.core.wait_strategies import HttpWaitStrategy
+
+
+# DockerSkippingContainer, KeepaliveContainer,
```

Comment:
This line looks like a leftover from the advanced variant we had been incubating in ctk. Please remove here?